CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ set(NFS_SOURCES src/NFSConnection.cpp
 
 set(DEPLIBS ${NFS_LIBRARIES} ${p8-platform_LIBRARIES} ${kodiplatform_LIBRARIES})
 
+if(WIN32)
+  list(APPEND DEPLIBS ws2_32)
```

Comment:
this is "all over". is it caused by the above libraries or nfs in this case?

Reply:
You are right, it should be moved to `FindNFS.cmake`.
Just didn't think about as I only wanted to get it compiled.

Reply:
right, so it is nfs and not the platform libraries. hoped we could kill more birds ;)

Reply:
fixed now